packages/fluentui/react/src/themes/teams/components/Card/cardBodyStyles.ts

```diff
@@ -0,0 +1,15 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { CardVariables } from './cardVariables';
+import { CardBodyStylesProps } from '../../../../components/Card/CardBody';
+
+const cardBodyStyles: ComponentSlotStylesPrepared<CardBodyStylesProps, CardVariables> = {
+  root: ({ variables: v, props: p }): ICSSInJSStyle => {
+    return {
+      display: 'flex',
+      flexDirection: 'column',
+      ...(!p.fitted && { marginBottom: v.cardChildMarginBottom })
```

Comment:
```suggestion
      marginBottom: v.marginBottom,
      ...(p.fitted && { marginBottom: 0 })
```

We want to override styles for special cases and have defaults in the beginning. This mimic the how css works. For the variables names, we don't need here the `cardChild` prefix, as card in the component itself and we are applying this on the root. If you were to add this on different slot, than the name would be: `[slot]MarginBottom`. 

Final comment, it's better to add one margin variable, instead of marginBottom, if it is necessary to override more things in the future.

Reply:
Good point, will update to use one margin variable